require_node.py

```diff
@@ -30,7 +30,8 @@ def write(index):
 
             region_to_insert = self.view.sel()[0]
 
-            line_is_empty = self.view.lines(region_to_insert)[0].empty()
+            line = self.view.substr(self.view.lines(region_to_insert)[0])
```

Comment:
that's to check whether line has withespaces (and actually is empty - try to insert require statement in function body - in such case there will be indentation). empty() method returns false in such cases (spaces, tabs)